src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -203,6 +205,13 @@ private void FindEnclosingExceptionRegions()
                             }
                         }
                     }
+                    if(r.FilterOffset != -1 && r.FilterOffset <= offset && r.HandlerOffset - 1 >= offset)
```

Comment:
Yes - from ECMA 335: "They nest: all three regions of one entry shall be within a single region of the other
entry, with the further restriction that the enclosing region shall not be a filter."